### PR TITLE
fix(core): fix no partition error after truncate and rollback

### PR DESCRIPTION
**Pull request by ideoma**
Fixes error 

```
2023-04-02T14:57:41.628661Z E i.q.t.TestListener ***** Test Failed *****io.questdb.test.griffin.wal.WalWriterFuzzTest.testWalWriteRollbackTruncateHeavy : 
io.questdb.cairo.CairoException: [0] Partition '2022-02-24' does not exist in table 'testWalWriteRollbackTruncateHeavy_nonwal' directory. Run [ALTER TABLE testWalWriteRollbackTruncateHeavy_nonwal DROP PARTITION LIST '2022-02-24'] to repair the table or restore the partition directory.
```

found in fuzz tests

### CODE REVIEW

**Comment by questdb-butler** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Please update the PR title to match this format:
`type(subType): description`

Where `type` is one of:
`feat`, `fix`, `chore`, `docs`, `style`, `refactor`, `perf`, `test`, `ci`, `chore`, `revert`

And: `subType` is one of:
`build`, `sql`, `log`, `mig`, `core`, `ilp`, `pgwire`, `http`, `conf`, `ui`, `wal`

For Example:

```
perf(sql): improve pattern matching performance for SELECT sub-queries
```
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against c743b4d0f96b249718e9475870bb571f0b226b56
</p>

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 4 / 4 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|4|4|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 17 / 18 (94.44%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/DefaultCairoConfiguration.java|1|2|50.00%|
|:large_blue_circle:|io/questdb/PropServerConfiguration.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/ServerMain.java|8|8|100.00%|
|:large_blue_circle:|io/questdb/mp/SOCountDownLatch.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/PropertyKey.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|4|4|100.00%|

**Review by puzpuzpuz — APPROVED** [review]

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 17 / 18 (94.44%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/DefaultCairoConfiguration.java|1|2|50.00%|
|:large_blue_circle:|io/questdb/PropServerConfiguration.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/ServerMain.java|8|8|100.00%|
|:large_blue_circle:|io/questdb/mp/SOCountDownLatch.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/PropertyKey.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|4|4|100.00%|

File: core/src/main/java/io/questdb/PropServerConfiguration.java [hunk 1]
```diff
@@ -474,6 +475,7 @@ public PropServerConfiguration(
         this.walTxnNotificationQueueCapacity = getQueueCapacity(properties, env, PropertyKey.CAIRO_WAL_TXN_NOTIFICATION_QUEUE_CAPACITY, 4096);
         this.walRecreateDistressedSequencerAttempts = getInt(properties, env, PropertyKey.CAIRO_WAL_RECREATE_DISTRESSED_SEQUENCER_ATTEMPTS, 3);
         this.walSupported = getBoolean(properties, env, PropertyKey.CAIRO_WAL_SUPPORTED, true);
+        walApplyEnabled = getBoolean(properties, env, PropertyKey.CAIRO_WAL_APPLY_ENABLED, true);
```
[the remark this answers is not in the document]
Comment (ideoma) [reply core/src/main/java/io/questdb/PropServerConfiguration.java:478]: Will add in another PR.

**Pull request merged**